.gitignore

```diff
@@ -10,6 +10,7 @@
 .pytest_cache
 .tox
 .vscode
+.venv*
```

Comment:
Why did you add that?

Reply:
A common pattern for virtual env directory? Must I remove it here?

Reply:
I don't think anyone will complain. I was just unsure because I never had to use that.

Reply:
HaHa. Check here: [part of default `.gitignore` file for Python projects on Github](https://github.com/github/gitignore/blob/4488915eec0b3a45b5c63ead28f286819c0917de/Python.gitignore#L122-L129).

Never mind, I'm fine to remove it.

Reply:
Oh well, just didn't create my venv in the very same directory as the code. But keep the ignore pattern for more such cases.